components/application-mgt/org.wso2.carbon.identity.application.common/src/main/java/org/wso2/carbon/identity/application/common/util/IdentityApplicationManagementUtil.java

```diff
@@ -740,4 +731,22 @@ public static String getDigestAlgoURIByConfig() {
         }
         return propValueSet;
     }
+
+    public static String getPropertyValue(Property[] properties, String propertyName) {
```

Comment:
There is already a method called getProperty() in same class. You just need to do getValue on that. We don't need a whole new method for that. However do the improvements in this method to the old method itself.